<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(34,34,34);
color:white;</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>254</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="about">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>260</width>
     <height>97</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="plainText">
    <string>Hello, World!</string>
   </property>
  </widget>
  <widget class="QLabel" name="_remind_me">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Remind Me</string>
   </property>
  </widget>
  <widget class="QLabel" name="_about">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bExit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>215</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bStop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="_after">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>After (hh:mm:ss)</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeAfter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
